src/main/java/core/basesyntax/service/AmountCalculator.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Operation;
+import java.util.List;
+import java.util.Map;
+
+public interface AmountCalculator {
+    Map<String, Integer> calculate(List<Operation> operations);
```

Comment:
Calculate is a very generic method name. Much more it returns some map, not long or int value. Can we rename this method?

Reply:
renamed to calculateDataForReport